schema/vrs/json/SequenceReference

```diff
@@ -5,7 +5,10 @@
    "type": "object",
    "maturity": "draft",
    "ga4ghDigest": {
```

Comment:
I don't love the name for this, but that can be another discussion for a later time. I think digest in the name can be confusing since value objects do not compute ga4gh digests.